README.md

```diff
@@ -1,51 +1,83 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Sparkbox Team Availability
 
 ## Getting Started
 
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-```
-
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
-
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
-
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
-
-
-
----
-
-## Browser Testing
+1. Clone the repo to your local machine.
+1. Install all required dependencies `npm i`.
+1. Start the development development server `npm run dev`.
+1. Project is now running at [http://localhost:3000](http://localhost:3000) 
+
+## Development 
+
+### Available Scripts
+
+| Script | Description |
+| ------- | ------- |
+| `npm run dev` | Run the development server |
+| `npm test` | Run all tests |
+| `npm run lint` | Lint all JavaScript and Sass/CSS |
+| `npm run lint:js` | Lint only JavaScript |
+| `npm run lint:css` | Lint all Sass/CSS |
+| `npm build` | Build the project for production |
```

Comment:
This would need to be `npm run build`.  It's annoying how `npm test` and `npm start` are valid, but other equally common script names aren't (feels like `lint` and `build` should have been added by now).